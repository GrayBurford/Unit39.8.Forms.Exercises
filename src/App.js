import logo from './logo.svg';
import './App.css';
// import BoxList from './ColorBoxMaker/BoxList';
import TodoList from './TodoApp/TodoList';

function App() {
  return (
    <div className="App">
      <h1>Todo List with React</h1>
      <TodoList />
    </div>
  );
}

export default App;
