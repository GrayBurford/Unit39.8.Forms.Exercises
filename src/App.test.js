import { render, screen } from '@testing-library/react';
import App from './App';



it('Renders App component without crashing', function () {
  render(<App />)
})

it('matches snapshot', () => {
  const { asFragment } = render(<App />)
  expect(asFragment()).toMatchSnapshot();
})