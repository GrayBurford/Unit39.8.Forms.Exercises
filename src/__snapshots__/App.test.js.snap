// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Todo List with React
    </h1>
    <div
      class="TodoList"
    >
      <div
        class="TodoList-form"
      >
        <form
          class="NewTodoForm"
        >
          <label
            for="task"
          >
            Add task
          </label>
          <input
            id="task"
            name="task"
            placeholder="type to-do here"
            type="text"
            value=""
          />
          <button>
            Submit To-Do
          </button>
        </form>
      </div>
      <div
        class="TodoList-todos"
      />
    </div>
  </div>
</DocumentFragment>
`;
